src/views/mod/customavatar.ejs

```diff
@@ -169,22 +170,39 @@
 
 
       function processAvatarRequest(decision, avatarreqid, comment) {
-        $.ajax({
-          url: '/profile/mod/ajax/processavatarrequest',
-          type: 'POST',
-          data: {
-            decision: decision,
-            avatarreqid: avatarreqid,
-            modcomment: comment,
-          },
-          success: function() {
-            console.log('ajax success');
-          },
-          error: function(err) {
-            console.log(err);
-          },
-        });
-
+          Swal.fire({
+              title: 'Sending your request...',
+              onOpen: () => {
+                  Swal.showLoading();
+                  axios({
+                      method: 'POST',
+                      url: '/profile/mod/ajax/processavatarrequest',
+                      data: {
+                          decision: decision,
+                          avatarreqid: avatarreqid,
+                          modcomment: comment,
+                      },
+                  })
+                      .then(function (response) {
+                          //handle success
+                          console.log('ajax success');
+
+                          Swal.close();
+                          Swal.fire({
+                              title: response.data,
+                              type: 'success',
+                          });
+                      })
+                      .catch(function (err) {
+                          console.log(err)
```

Comment:
rm

Reply:
Do we not want the logs for this one anymore?